guides/create-graphql-query.md

```diff
@@ -0,0 +1,232 @@
+# How To: Create a new GraphQL query
+
+## Step 1: Identify which plugin owns the query
+
+The complete Reaction Commerce GraphQL API is created by stitching together domain-specific APIs from all of the API plugins. So when adding a new query, the first step is to decide which plugin should own it. This is usually obvious, but not always. You should think about whether any other plugins or services will need to call your query. If the query is fundamental to the system, then it may need to go in the "core" plugin, if no better alternative exists.
+
+## Step 2: Understand the difference between a plugin query function and a GraphQL query resolver
+
+See [Resolver Mutations and Queries vs. Plugin Mutations and Queries](../guides/developers-guide/core/developing-graphql.md#resolver-mutations-and-queries-vs-plugin-mutations-and-queries)
```

Comment:
this concept is now in the graphql-concepts doc
https://github.com/reactioncommerce/docs/pull/56/files line 70

Reply:
Yes, the concept is, but this is a guide that walks you through creating it